template/elixir/connection.ex.eex

```diff
@@ -20,6 +20,8 @@ defmodule <%= namespace %>.Connection do
   Handle Tesla connections for <%= namespace %>.
   """
 
+  @type t :: Tesla.Env.client()
```

Comment:
Is this accurate? Even if so, we may want be revisiting this interface when implementing #98

Reply:
Yes, it is accurate as it stands. (In fact, in the API endpoint calls where this is used, the `@spec` actually lists `Tesla.Env.client()` although the documentation wants to use the connection module, hence my addition of this typespec.)

It turns out that there are currently three spellings of this same type (`Tesla.Client.t()`, `Tesla.Env.client()`, and `Path.To.API.Connection.t()`) and each of the three is used in different places, rather inconsistently. We should clean that up, but, as you say, it might make sense to do that in conjunction with #98.